packages/arm-core/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@datapunt/arm-core",
+  "description": "",
+  "license": "MPL-2.0",
+  "author": "Datapunt Amsterdam",
+  "homepage": "https://github.com/Amsterdam/amsterdam-react-maps#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Amsterdam/amsterdam-react-maps.git",
+    "directory": "packages/arm-core"
+  },
+  "bugs": {
+    "url": "https://github.com/Amsterdam/amsterdam-react-maps/issues"
+  },
+  "version": "0.2.7",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "files": [
+    "es",
+    "lib",
+    "static"
+  ],
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "sideEffects": false,
+  "typings": "lib/index.d.ts",
+  "dependencies": {
+    "geojson": "^0.5.0",
+    "proj4": "^2.6.2",
+    "proj4leaflet": "^1.0.2",
+    "uuid": "^8.1.0",
+    "leaflet": "^1.6.0"
+  },
+  "peerDependencies": {
+    "@datapunt/asc-assets": "^0.20.0",
+    "@datapunt/asc-ui": "^0.20.0",
+    "@datapunt/react-maps": "^0.8.2-alpha.0",
+    "react": ">=16.8.6",
+    "react-dom": ">=16.8.6",
+    "styled-components": ">=4.4.1",
+    "leaflet": "^1.6.0"
```

Comment:
If this is a peer dependency, shouldn't it also be listed in dev dependencies instead of dependencies?

Reply:
It's already in the root package.json